### PR TITLE
MWS-14 About our team name changed

### PR DESCRIPTION
**Pull request by Sarath-VC**
@narayananperumba @manojmedha sample workout of jira task.

### CODE REVIEW

File: resources/views/aboutteam.blade.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
                             </div>
                             <div class="dlab-content">
                                 <div class="clearfix">
-                                    <h3 class="dlab-name"><a href="javascript:void(0);">John Doe</a></h3>
+                                    <h3 class="dlab-name"><a href="javascript:void(0);">Manoj</a></h3>
                                     <span class="dlab-position">Senior Designer</span>
```
Comment (manojmedha) [inline resources/views/aboutteam.blade.php:34]: Considering this is a test, when we update it to actual work, will have to update this with relevant info

File: debug.log [hunk 2]
```diff
@@ -0,0 +1 @@
+[0122/104722.476:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (manojmedha) [inline debug.log:1]: pl. remember to delete this if not required (it is a debug log, so can be removed)

**Review by manojmedha — APPROVED** [review]

**Comment by medhaadmin** [comment]
We need a generic view, will give some design soon

**Comment by medhaadmin** [comment]
changes are moved to another PR

**Pull request closed without merging**